.github/workflows/leo-build-tag-publish-and-run-tests.yml

```diff
@@ -182,6 +197,32 @@ jobs:
               "custom-version-json": "${{ steps.render-leo-version.outputs.custom-version-json }}"
             }'
 
+  azure-automation-tests:
+    runs-on: ubuntu-latest
+    # TODO: fix the  `ref`
```

Comment:
what needs to be done to fix the `ref`?

Reply:
Ah, I had to hardcode the ref here to be this branch since the changes to the workflow file were on this branch, now it points to develop again correctly. The TODO can go now.

There already exists a way to run azure automation tests against a specific branch with a separate param for the branch the workflow file to run the tests is sourced from, but this change will mean any PR that runs azure automation tests will take the workflow file to run the tests from develop (and the app code and test code will be from the PR branch build).  

tl;dr Very confusing since there are **three** relevant branches:
- the branch the workflow file running the test code is from
- the branch the automation code running in the GHA node is from
- the branch the leo code deployed to a bee is from